lawnchair/src/app/lawnchair/search/data/SearchRequest.kt

```diff
@@ -36,53 +36,57 @@ private val retrofit = Retrofit.Builder()
 val startPageService: StartPageService = retrofit.create()
 
 suspend fun getStartPageSuggestions(query: String, max: Int, callback: SearchCallback) {
-    try {
-        if (query.isEmpty() || query.isBlank() || max <= 0) {
-            callback.onSearchLoaded(emptyList())
-            return
-        }
+    val exceptionHandler = CoroutineExceptionHandler { _, e ->
+        Log.e("Exception", "Error during suggestion retrieval: ${e.message}")
```

Comment:
Have you tried this without data connection? I intended to surround it in try catch block as it's crashing when no data connection

Reply:
May need to add `SupervisorJob()`.

Reply:
Please revert back surrounded try catch block thanks